static/styles/_base/_tables.scss

```diff
@@ -13,9 +15,8 @@ table {
   }
 
   th {
-    padding: 1rem;
-    height: 4rem; 
-    border-bottom: 1px solid $light-gray;
+    padding: 1rem !important;
```

Comment:
I think this should be restructured so `!important` isn't required, although I think Noah is helping with this.